cmd/trillian_map_server/main.go

```diff
@@ -40,7 +40,6 @@ import (
 	"github.com/google/trillian/server"
 	"github.com/google/trillian/storage"
 	etcdutil "github.com/google/trillian/util/etcd"
-	"github.com/grpc-ecosystem/grpc-gateway/runtime"
 	"google.golang.org/grpc"
 
 	// Register key ProtoHandlers
```

Comment:
The httpEndpoint flag says "Endpoint for HTTP metrics and **REST requests**" - should that second part be removed now?